src/test/scala/com/databricks/labs/overwatch/pipeline/InitializeTest.scala

```diff
@@ -0,0 +1,224 @@
+package com.databricks.labs.overwatch.pipeline
+
+import com.databricks.labs.overwatch.SparkSessionTestWrapper
+import com.github.mrpowers.spark.fast.tests.DataFrameComparer
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.PrivateMethodTester
+import com.databricks.labs.overwatch.utils.{BadConfigException, Config, OverwatchScope}
+import com.databricks.labs.overwatch.env.Database
+import com.databricks.labs.overwatch.utils.OverwatchScope._
+
+class InitializeTest extends AnyFunSpec with DataFrameComparer with SparkSessionTestWrapper with PrivateMethodTester {
+
+  describe("Tests for Initializer.isPVC") {
+    it("should validate isPVC as false when org id if doesn't have ilb") {
+      val conf = new Config
+      conf.setOrganizationId("demo")
+      val init = new Initializer(conf)
+
+      val isPVC = PrivateMethod[Boolean]('isPVC)
+      val actual = init invokePrivate isPVC()
+      println("value: " + actual)
+      assert(!actual)
+    }
+  }
```

Comment:
I think we should add a positive test case as well. i.e. where orgID has ilb in it